pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -431,7 +405,7 @@ func (e *TokensController) ensureReferencedToken(serviceAccount *v1.ServiceAccou
 	}
 	// Manually add the new token to the cache store.
 	// This prevents the service account update (below) triggering another token creation, if the referenced token couldn't be found in the store
-	e.secrets.Add(createdToken)
+	e.secretStore.Add(createdToken)
```

Comment:
I will fix it ... somehow. Maybe just revert back to using a separate store for secrets here for now.